src/resources/base.js

```diff
@@ -143,6 +144,9 @@ export default api => (
           res = await api[method](url);
         }
 
+        if (pathsWithoutEasyPostObjects.includes(path)) {
```

Comment:
# Reviewer Note

This was done because we don't want to append the `smartrates` results to the `Shipment` object, we only want the `results`. This list can be extensible if we had other methods that needed to take advantage in the future.